src/Symfony/Component/Config/Tests/Util/XmlUtilsTest.php

```diff
@@ -55,7 +55,7 @@ public function testLoadFile()
             XmlUtils::loadFile($fixtures.'valid.xml', array($mock, 'validate'));
             $this->fail();
         } catch (\InvalidArgumentException $e) {
-            $this->assertRegExp('/The XML file "[\w:\/\\\.~+-]+" is not valid\./', $e->getMessage());
+            $this->assertRegExp('/The XML file "[\w:\/\\\.~+-]+" is not valid\./u', $e->getMessage());
```

Comment:
This works with non-utf8 paths too

Reply:
Where does the UTF8 char comes from, that lead to this change?
I suppose it comes from your directory layout, that contains UTF-8 chars in the parent dir, is that the case?
Then if anyone else has a parent directory that containes a high-ascii characters encoded in anything else than UTF-8, then this will break for them.
I'd put `.+` in fact instead of the regexp class (which provide nothing special, isn't it?)